### PR TITLE
Add texture

### PR DESCRIPTION
**Pull request by 0xsuu**

### CODE REVIEW

**Comment by 0xsuu** [comment]
@ZongyuYin review

File: src/ABETextLabel.cpp [hunk 1]
```diff
@@ -17,95 +17,146 @@
...
+    glActiveTexture(GL_TEXTURE0);
+//    glEnableVertexAttribArray(0);
+    glBindVertexArray(mVertexArrayID);
+//    glVertexAttribPointer(0, 4, GL_FLOAT, GL_FALSE, 4 * sizeof(GLfloat), 0);
+
+    // Iterate through all characters.
+    float x = getX();
+    float y = getY();
+    std::string::const_iterator c;
+    for (c = mText.begin(); c != mText.end(); c++) {
```
Comment (0xsuu) [inline src/ABETextLabel.cpp]: @ZongyuYin Gai

File: external/CMakeLists.txt [hunk 2]
```diff
@@ -38,3 +40,9 @@ add_library(GLEW_200 STATIC
 target_link_libraries(GLEW_200
         ${OPENGL_LIBRARY}
         ${EXTRA_LIBS})
+
+add_subdirectory(harfbuzz-1.4.6)
```
Comment (0xsuu) [inline external/CMakeLists.txt]: Remove this line.

File: external/CMakeLists.txt [hunk 3]
```diff
@@ -38,3 +40,9 @@ add_library(GLEW_200 STATIC
...
+
```
Comment (0xsuu) [inline external/CMakeLists.txt]: Remove this line.

**Review by Nniy — APPROVED** [review]
Perfect

**Pull request merged**